community/tools/github/dco-and-licenses.md

```diff
@@ -44,7 +44,7 @@ line to commit messages.
 ```bash
 My fancy commit message
 
-Signed-off-by: Christian Berendt <berendt@betacloud-solutions.de>
+Signed-off-by: Christian Berendt <$berendt@betacloud-solutions.de>
```

Comment:
I can live fine with most changes, but this seems wrong.

Reply:
![Screenshot from 2024-04-02 17-28-28](https://github.com/SovereignCloudStack/docs/assets/11217413/cf994466-55bd-4b0d-9e4c-d5ecc219ac55)
No, this is a code segment, so the escape character will not be interpreted. Why did you put it in there?